cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsInstanceConnector.java

```diff
@@ -185,6 +188,28 @@ private void doStart(List<CloudInstance> affectedVMs, AuthenticatedContext ac, L
         }
     }
 
+    private void doStart(List<CloudInstance> affectedVMs, AuthenticatedContext ac, List<CloudInstance> instances, List<CloudVmInstanceStatus> rebootedVmsStatus) {
```

Comment:
why did you introduce a new `doStart`? the old one is not used anymore

Reply:
The reason to introduce new doStart() is we want to return the polling results from doStart(). The polling will take 1000 seconds total. In the case of a hang blocking call, we will wait twice as much long. It is not necessary to poll inside doStart() and then after return from doStart() immediately poll again.